plugins/chat/assets/javascripts/discourse/components/chat/composer/thread.js

```diff
@@ -2,17 +2,24 @@ import ChatComposer from "../../chat-composer";
 import { inject as service } from "@ember/service";
 import I18n from "I18n";
 import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
 
 export default class ChatComposerThread extends ChatComposer {
   @service("chat-channel-thread-composer") composer;
   @service("chat-channel-composer") channelComposer;
   @service("chat-channel-thread-pane") pane;
   @service router;
 
+  @tracked thread;
```

Comment:
Is this needed? Looks like thread is being passed in as an argument instead?

Reply:
Oh yes forgot to remove it thanks :+1: